core/src/consensus/stake/mod.rs

```diff
@@ -860,7 +884,7 @@ mod tests {
         // TODO: change with stake.execute()
         self_nominate(&mut state, &address, &address_pubkey, 200, 0, 30, b"".to_vec()).unwrap();
 
-        let result = on_term_close(&mut state, 29);
+        let result = on_term_close(&mut state, pseudo_term_to_block_num_calculator(29));
```

Comment:
`pseudo_term_to_block_num_calculator` is somewhat misleading. Actual bulk term increasing thing is done with `increase_term_id_until`, `on_term_close` is increasing it just one step. I think turn `increase_term_id_until` into `term_close_until` or something

Reply:
It doesn't increase the term bulky.
The second parameter of `on_term_close` is the block number of closing block.
And you cannot rename `increase_term_id_until` to `term_close_until`, because it doesn't close term, it merely increases term id.

Reply:
I understand it now that `pseudo_term_to_block_num_calculator` is not actually related term id. It was confusing for a while.
I think increasing term id by closing the term is more realistic than calling `increase_term_id` directly.

Reply:
It is the purpose of this function to set the states for testing not calling `on_term_close`.